tests/models/encoder_decoders/wikitables_semantic_parser_test.py

```diff
@@ -305,6 +305,15 @@ def test_map_entity_productions(self):
                 }
 
 
+class WikiTablesSemanticParserTestWithMixtureFeedforward(ModelTestCase):
+    def setUp(self):
+        super(WikiTablesSemanticParserTestWithMixtureFeedforward, self).setUp()
+        self.set_up_model("tests/fixtures/encoder_decoder/wikitables_semantic_parser_with_mixture_feedforward/experiment.json", # pylint: disable=line-too-long
```

Comment:
Ok, last nitpick, then I think this is good to merge.  Can you move this line inside the `test_model...` method, then move that method inside the already-existing class?  There isn't really a reason to create a new class for this.

Great work on this!